### ISSUE TITLE
Which pixel data range?

### ISSUE THREAD

**Issue by done1892** [mlmed/torchxrayvision#10]
Hi. Firstly, thank you for sharing.
I'm using your weights to do finetuning and train the network, with additional fully connected layers, on my own dataset. Which normalization have you used to train your network? I suppose the data are in [0, 1] pixel range.

**Comment by ieee8023** [comment]
Here: https://github.com/mlmed/torchxrayvision/blob/master/torchxrayvision/datasets.py#L47
